sonata-project/classification-bundle/3.7/src/Entity/SonataClassificationCollection.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as Serializer;
+use Sonata\ClassificationBundle\Entity\BaseCollection;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="classification__collection")
```

Comment:
Do you really want the double underscore?

Reply:
This name appeared in older versions (i do not even know when). It will be better not to change it, just not to confuse developers. Also with double underscore these tables are groupped in phpmyadmin =)

Reply:
Great great great.